src/test/java/com/google/api/codegen/testdata/discoveries/go/go_deploymentmanager.v2.json.baseline

```diff
@@ -1033,9 +1033,9 @@ func main() {
 
   req := deploymentmanagerService.Types.List(project)
   if err := req.Pages(ctx, func(page *deploymentmanager.TypesListResponse) error {
-    for _, type := range page.Types {
-      // TODO: Change code below to process each `type` resource:
-      fmt.Printf("%#v\n", type)
+    for _, type2 := range page.Types {
+      // TODO: Change code below to process each `type2` resource:
```

Comment:
This comment seems wrong, anyway.

Shouldn't it say

`// TODO: Change code below to process each `Type` resource:`

Lowercase "type" is the variable name, not the name of the type of resource. Confusing in this case, since the type is named `Type`.